### PR TITLE
remove unused variable

### PR DESCRIPTION
**Pull request by RafaelVianaCunha**
I just removed an unused variable

### CODE REVIEW

**Comment by RafaelVianaCunha** [comment]
@minhphuc429 
Okay, could you please look again?

File: modules/ta.js [hunk 1]
```diff
@@ -36,12 +36,11 @@ module.exports = class Ta {
                                 return;
                             }
 
-                            let range = moment().subtract(24, 'hours')
+                            //let range = moment().subtract(24, 'hours')
```
Comment (Haehnchen) [inline modules/ta.js]: feel free to directly remove this line ;)

Comment (RafaelVianaCunha) [reply modules/ta.js]: OK

**Comment by Haehnchen** [comment]
did a rewrite of this function to speed up sqlite3, this line was also removed

**Pull request closed without merging**